src/Icicle/Avalanche/Statement/Simp/Constructor.hs

```diff
@@ -80,7 +80,7 @@ constructor a_fresh statements
 
   goS env s
    = let env' = updateExpEnv s env
-         go   = goX env
+         go   = goX env'
```

Comment:
@amosr I think there was a missing prime here. It'll make constructor less effective (i.e. needs to run more constructor passes that otherwise needed)